scripts/upload_ceph_images_to_nexus.sh

```diff
@@ -312,11 +312,89 @@ function exit_maintenance_mode() {
   fi
 }
 
+function disable_local_registries() {
+  echo "Disabling local docker registries"
+  systemctl_force="--now"
+
+  for storage_node in $(ceph orch host ls -f json |jq -r '.[].hostname'); do
+    if ssh ${storage_node} ${ssh_options} "systemctl disable registry.container.service ${sysetmctl_force}"; then
+       if ! ssh ${storage_node} ${ssh_options} "systemctl is-enabled registry.container.service"; then
+         echo "Docker registry service on ${storage_node} has been disabled"
+       fi
+    fi
+  done
+}
+
+function fix_registries_conf() {
+
+  for storage_node in $(ceph orch host ls -f json |jq -r '.[].hostname'); do
```

Comment:
This creates a here file each time through. Probably simpler to just create it once and then loop (by moving for down a tad bit). We can also probably use something smoother than cat-ing that file into an ssh (like scp, rsync, etc). Probably be easier to follow that way.
Where is this running from (I presume one of the storage nodes and then it updates that same said storage node. Might be easier to skip the current node.)